### PR TITLE
Improve instructions about setting GMT_LIBRARY_PATH env variable

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Fixes https://github.com/GenericMappingTools/pygmt/issues/1993

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -259,12 +259,17 @@ library (``libgmt``).
 This can happen if you have multiple versions of GMT installed.
 
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
-``GMT_LIBRARY_PATH`` environment variable.
-This should be set to the directory where ``libgmt.so``, ``libgmt.dylib`` or
-``gmt.dll`` can be found for Linux, macOS and Windows, respectively.
-e.g., on a command line, run::
+``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,
+``libgmt.dylib`` or ``gmt.dll`` can be found for Linux, macOS and Windows,
```
Comment (weiji14) [inline doc/install.rst:263]:
```suggestion
``libgmt.dylib`` or ``gmt.dll`` can be found on Linux, macOS or Windows,
```

File: doc/install.rst [hunk 2]
```diff
@@ -259,12 +259,17 @@ library (``libgmt``).
...
+respectively.
+
+For Linux/macOS, adding the following line to your shell configuration file
+(usually `~/.bashrc` for Bash on Linux and `~/.zshrc` for Zsh on macOS)::
```
Comment (weiji14) [inline doc/install.rst:267]: Should we use the tab thing for Linux/macOS/Windows instructions? Similar to what is currently used for conda/mamba?

File: doc/install.rst [hunk 3]
```diff
@@ -259,12 +259,17 @@ library (``libgmt``).
...
 
-    # Linux/macOS
     export GMT_LIBRARY_PATH=$HOME/anaconda3/envs/pygmt/lib
-    # Windows
-    set "GMT_LIBRARY_PATH=C:\Users\USERNAME\Anaconda3\envs\pygmt\Library\bin\"
+
+For Windows, adding the ``GMT_LIBRARY_PATH`` environment variable following the
```
Comment (weiji14) [inline doc/install.rst:271]:
```suggestion
For Windows, add the ``GMT_LIBRARY_PATH`` environment variable following these
```

File: doc/install.rst [hunk 4]
```diff
@@ -260,15 +260,15 @@ This can happen if you have multiple versions of GMT installed.
 
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
 ``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,
-``libgmt.dylib`` or ``gmt.dll`` can be found for Linux, macOS and Windows,
+``libgmt.dylib`` or ``gmt.dll`` can be found on Linux, macOS or Windows,
 respectively.
 
 For Linux/macOS, adding the following line to your shell configuration file
```
Comment (seisman) [inline doc/install.rst:266]:
```suggestion
For Linux/macOS, add the following line to your shell configuration file
```

File: doc/install.rst [hunk 5]
```diff
@@ -259,12 +259,17 @@ library (``libgmt``).
...
-``GMT_LIBRARY_PATH`` environment variable.
-This should be set to the directory where ``libgmt.so``, ``libgmt.dylib`` or
-``gmt.dll`` can be found for Linux, macOS and Windows, respectively.
-e.g., on a command line, run::
+``GMT_LIBRARY_PATH`` environment variable to the directory where ``libgmt.so``,
+``libgmt.dylib`` or ``gmt.dll`` can be found on Linux, macOS or Windows,
+respectively.
+
+For Linux/macOS, add the following line to your shell configuration file
+(usually `~/.bashrc` for Bash on Linux and `~/.zshrc` for Zsh on macOS)::
```
Comment (seisman) [inline doc/install.rst:267]:
```suggestion
(usually ``~/.bashrc`` for Bash on Linux and ``~/.zshrc`` for Zsh on macOS)::
```

File: doc/install.rst (hunk 2 above)
Comment (seisman) [reply doc/install.rst]: I don't think it's necessary.

**Review by weiji14 — APPROVED** [review]

**Pull request merged**